CONTRIBUTING.md

```diff
@@ -1,41 +1,43 @@
 # How To Contribute
-Contributing to Gaia Dimension has various ways to contribute to the development. From a standpoint, contributions can include Coding, Assets and Testing. Please note that any PRs made can and will be rejected if they do not comply with the following guidelines. PRs will be closed if:
+There are various ways to contribute to the development of the Gaia Dimension mod. Some common types of contributions can include donating code, making textures, translating, and testing. Please note that any Pull Requests (PRs) made can and will be rejected if they do not comply with the following guidelines.
 
-* Does not comply with the listed guielines
-* No action is done to resolve conflicts or build failures within 2 weeks
-
-## Coding
-Though it is not easy from a surface level, it's worth more to contribute via coding than any other place. However, there are a few things to note when submitting a PR for coding.
+## Code
+Though it is not easy, donating code is the best way to contribute. However, there are a few things to note when submitting a PR with code.
 
 ### Make sure braces are on *the same line*
 As much as some people are inclined to, and for consistency's sake, make sure your code is formatted to use same-line braces. This is to reduce on file size and better vertical readability.
 
 Example:
-```
+```java
 public void() {
 }
 ```
 ### It needs to work
 You can submit a PR that's got a solution to a problem or a new feature, but it needs to build. This can be checked by submitting the PR's and looking at the mergability. This repo uses Travis-CI, so builds are often checked.
 That being said, it also needs to be functional in-game. You could submit a PR based on the grounds that it works, but thorough testing is required to make sure that it doesn't break any other feature.
 
-## Assets
-Though they are fundamental, contributing to these isn't as desired due to how easy it is to contribute this way. Just like Coding, there are a few guidelines that need to be followed.
+## Textures
+Though they are fundamental, contributing to these isn't as desired by developers due to how easy it is to contribute this way. Just like coding, however, there are a few guidelines that need to be followed.
 
 ### Please try to submit the JSON for the texture you wish to add
 If something is implemented but untextured, you may wish to add a texture to those. However, it is most desirable if a JSON is submitted this way too. Unless specified, a PR is unlikely to be pulled if a texture is added but no JSON is attached.
 
-### Regarding JSONS, use Forge Blockstates
```

Comment:
I would just remove this section. Forge Blockstates was a 1.12 thing that was then removed post-1.13, and Gaia Dimension only supports the latest version - 1.16 in this case.

Instead, I would change it to requiring blockstates and models to be generated via Data Generators where applicable.
"Use Data Generators for Blockstates and Models where possible: To save on time and energy, most blocks and items have generated files where they are output to `src/generated`. Unless the block or item has a unique model that would take more time to write a generator for, please use the Data Generators provided in `src/main/java/androsa/gaiadimension/data/`."